nkalyesubula/AndaApp
# Ft api get blog 167449792

### What does this PR do?
  Add the view single blog feature
### Description of Task to be completed?

  - Add the blog view route
  - Add the view controller function with integrated target existence check

### How should this be manually tested?

  - Run the command git checkout -b ft-api-get-blog-167449792
  - Run git pill ft-api-get-blog-167449792
  - On postman, run a GET/http://localhost:3000/api/article/2 request
    Change the id value to a large value, say 100000 and see the 404 response
  - Alternatively, open your browser and type localhost:3000/api/article/2 in the address bar

### Any background context you want to provide?

### What are the relevant pivotal tracker stories?

  [finishes #167449792](https://www.pivotaltracker.com/story/show/167449792)

c45979a
### Screenshots (if appropriate)

### Questions:
